### ISSUE TITLE
EasyDataManagerEF fails with entities that are generic types

### ISSUE THREAD

**Issue by chaghi** [KorzhCom/EasyData#49]
I'm trying to use EasyData in a project that has the "Individual Accounts" feature enabled, i.e., it has the ASP.NET Identity data model. Several of the entities won't load, and the culprit seems to be the generic types (`IdentityUserClaim<TKey>`, `IdentityUserRole<TKey>`, etc.)

When `EasyDataManagerEF` tries to find the entity type, it fails:

```csharp
        private IEntityType GetCurrentEntityType(DbContext dbContext, string entityContainer)
        {
            var entityType = dbContext.Model.GetEntityTypes()
                .FirstOrDefault(ent => 
                    ent.Name.Split('.').LastOrDefault() == entityContainer);

            if (entityType == null) {
                throw new ContainerNotFoundException(entityContainer);
            }

            return entityType;
        }
```

In this scenario, `entityContainer` is for example `IdentityRoleClaim`, while the model's entity type is `IdentityRoleClaim<int>`.

As a quick test/workaround, I implemented my own `EasyDataManager` based on `EasyDataManagerEF`, and these two changes seem to do the trick:

```diff
diff --git a/easydata.net/src/EasyData.EntityFrameworkCore.Relational/Services/EasyDataManagerEF.cs b/easydata.net/src/EasyData.EntityFrameworkCore.Relational/Services/EasyDataManagerEF.cs
index 55f2075..6c96d56 100644
--- a/easydata.net/src/EasyData.EntityFrameworkCore.Relational/Services/EasyDataManagerEF.cs
+++ b/easydata.net/src/EasyData.EntityFrameworkCore.Relational/Services/EasyDataManagerEF.cs
@@ -69,7 +69,7 @@ namespace EasyData.Services
             var result = new EasyDataResultSet();
 
             var modelEntity = Model.EntityRoot.SubEntities.FirstOrDefault(e => e.ClrType == entityType.ClrType);
-            var attrIdProps = entityType.GetProperties().ToDictionary(prop => DataUtils.ComposeKey(entityType.Name.Split('.').Last(), prop.Name), prop => prop );
+            var attrIdProps = entityType.GetProperties().ToDictionary(prop => DataUtils.ComposeKey(entityType.Name.Split('.').Last().Split('<').First(), prop.Name), prop => prop );
 
             var attrs = modelEntity.Attributes.Where(attr => attr.Kind != EntityAttrKind.Lookup);
             foreach (var attr in attrs) {
@@ -169,7 +169,7 @@ namespace EasyData.Services
         {
             var entityType = dbContext.Model.GetEntityTypes()
                 .FirstOrDefault(ent => 
-                    ent.Name.Split('.').LastOrDefault() == entityContainer);
+                    ent.Name.Split('.').LastOrDefault().Split('<').FirstOrDefault() == entityContainer);
 
             if (entityType == null) {
                 throw new ContainerNotFoundException(entityContainer);
```

The idea is the same in both places: `Split` by `<` and take the first element, so when the entity type is generic, the generic part is omitted.

I'm not sure about creating a PR, because I'm not familiarized with the code, and I'm not sure if this is the optimal solution. Also, this might be kind of a niche use case for EasyData.

## Steps to reproduce

1. Create a new project with "Individual accounts" (`dotnet new webapp --auth Individual`)
2. Setup EasyData
3. Browse any generic entity, for example `/easydata/IdentityRoleClaim`

## Expected result

An empty grid (as there are no records yet), with the option to add a new one.

## Actual result

EasyData fails, and the grid is not rendered. There's a `404` during the request to fetch data: `POST https://localhost:5001/api/easydata/models/EasyData/crud/IdentityRoleClaim/fetch`

**Comment by antifree** [comment]
Thank you for such a detailed description of the problem!

We will investigate this and publish the solution as 1.3.2

**Comment by antifree** [comment]
I think it is fixed a6851b0eea44ec32c00cace87f75bd4cac178a27. 
It would be cool if you could get the latest dev and check before we release it. As I can see you have set up the dev environment.

**Comment by chaghi** [comment]
Yes, it seems to be fixed. Thank you!

**Comment by antifree** [comment]
Fixed in version 1.3.2. PR #51
